app/assets/stylesheets/components/maps/_v-map-header.scss

```diff
@@ -0,0 +1,20 @@
+.v-map-header {
+  background-color: $grey-black;
+  color: $grey-xlight;
+  display: flex;
+  font-family: $header-font;
+  font-size: rem-calc(25);
+  font-weight: 600;
```

Comment:
Is there a font weight variable (bold or semi bold) that we can use here?

Reply:
Ah yes, I haven't noticed. Will use $semi-bold